test/lightning/impact_tracking/client_test.exs

```diff
@@ -0,0 +1,32 @@
+defmodule Lightning.ImpactTracking.ClientTest do
+  use ExUnit.Case, async: false
+
+  import Tesla.Mock
+
+  alias Lightning.ImpactTracking.Client
+
+  @host "https://foo.bar"
+
+  describe ".impact_tracker" do
+    test "submits supplied metrics to impact tracker" do
```

Comment:
the `describe` is used as a prefix on test results (reading the concatenation would sound better like "client submits supplied metrics to impact tracker")